_drafts/getting-started-guide/whats-spyrker-cloud-commerce-os.md

```diff
@@ -0,0 +1,56 @@
+description: Learn about Spryker Cloud Commerce OS
+last_updated: Aug 17, 2023
+template: concept-topic-template
+---
+
+*Spryker Cloud Commerce OS (SCCOS)* is an e-commerce platform-as-a-service solution designed to provide businesses with the flexibility and efficiency needed to create a unique digital commerce experience. Built on a modular and layered architecture, it increases operational efficiency and lowers the total cost of ownership. This document provides an overview of the key features and benefits of SCCOS.
+
+## Modular Architecture
+
+The Spryker Commerce OS adopts a modular architecture, comprising over 750 different modules, with some being mandatory and others optional. This design allows you to create tailored solutions that match specific business needs.
+
+![Modularity Image](https://spryker.s3.eu-central-1.amazonaws.com/docs/About/About+Spryker/modularity_transparent.png)
+
+### Advantages of modular architecture
+
+* **Selectivity**: Utilize only the modules you need, reducing redundant code.
+* **Scalability**: Grow your project with the wide selection of available modules.
+* **Flexibility**: Add, delete, and test new features without hindering your live shop.
+* **Atomic Release Approach**: Each module is developed and released independently, ensuring backward compatibility.
+
+[More on Spryker Modules](https://github.com/spryker)
+
+## Layered Architecture
+
+Layered Architecture in SCCOS ensures a clear separation between commercial offerings and sales channels. This architecture is split into four different layers:
+
+* **Presentation Layer**: Interfaces like online stores, mobile apps, etc.
+* **Business Layer**: Contains products, pricing, stock, and commercial information.
+* **Communication Layer**: Connects the presentation to the business layer.
+* **Persistence Layer**: Manages data storage and processing.
+
+![Layers Image](https://spryker.s3.eu-central-1.amazonaws.com/docs/About/About+Spryker/spryker_layers_s.png)
+
+### Advantages of layered architecture
+
+* **High Performance**: Long processes are confined to specific layers, ensuring optimal performance.
+* **Parallel Development**: Frontend and backend development can occur simultaneously.
+* **Growth Focus**: Efforts can concentrate on improvement rather than makeshift solutions.
+* **Information Flexibility**: Easy to modify, extend, or replace essential information.
+
+## Managed Cloud Platform
+
+Spryker Cloud Commerce OS offers a managed cloud platform where the environments are fully managed by Spryker. This lets you achieve the following goals:
+
+* **Focus on development**:  You concentrate on creating and refining your projects because you don't need to worry about managing environments.
+* **Integrate seamlessly**: With CI/CD development workflows, you can achieve a streamlined development process.
+
+<!-- Additional details about the managed cloud platform will be added here once obtained from the experts. -->
+
+## Customizable
```

Comment:
Spryker stands out for its exceptional customizability, allowing businesses to finely tailor their e-commerce platform to align perfectly with their unique needs and vision. Its modular architecture empowers companies to select, integrate, and innovate with a diverse range of components, ensuring agility and scalability. This level of adaptability translates to personalized user experiences, streamlined business processes, and the swift adoption of cutting-edge technologies. By offering a canvas for creativity and strategic customization, Spryker empowers businesses to create a distinctive online presence that drives engagement, conversions, and long-term success.

Reply:
@helen-laktionova Please keep in mind that we should avoid marketing slang in documentation

Reply:
I played a bit with the above messaging, adding some important "technical" ideas, and avoiding marketing slogans as much as possible. Here is what I was able to come up with:

Spryker Cloud Commerce OS offers an adaptable platform suitable for a variety of e-commerce needs, including various business operational models and industry verticals. This has been achieved due to one of its primary features - ability for development teams to modify vendor-specific business logic provided out of the box, aligning the platform with individual operational requirements. Built on a modular and layered structure, Spryker is designed for making specific adjustments. It's architecture allows for easier customisation to meet distinct business processes and the smooth integration with pre-existing service Eco-systems. You have also the autonomy to add or remove functionalities based on your current requirements and needs. This flexibility is beneficial for iterative development, enabling teams to launch with essential feature set (MVP approach) and then incrementally introduce additional functionalities as needed.

Reply:
As an option, it can be formatted into bullet-points as following:

**Adaptable Platform**: Spryker Cloud Commerce OS is suitable for a variety of e-commerce needs, accommodating various business operational models and industry verticals.

**Vendor Logic Customisation**: Development teams can modify vendor-specific business logic provided out of the box leveraging Spryker's modular and layered architecture. This design allows for tailored adjustments to align with individual business requirements.

**Smooth Integration**: Spryker's architecture facilitates seamless integration with pre-existing service eco-systems.

**Iterative Development & Autonomy**: With Spryker, teams can benefit from the flexibility of an MVP approach, bringing their solutions fast to the market. They can start with essential features and integrations, and progressively introduce or remove functionalities, adjusting to evolving requirements.

Reply:
cc @VadymSachenko  @helen-laktionova

Reply:
Looks good, @VadymSachenko what do you think?